<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="file_container">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>581</width>
      <height>541</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="files_section">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>562</width>
       <height>539</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </widget>
   <widget class="QGroupBox" name="operations_container">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>191</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>Operations</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="upload_files_button">
       <property name="text">
        <string>Upload File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="uploading_files_status">
       <property name="title">
        <string>Uploaidng FIles</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="downloading_files_status">
       <property name="title">
        <string>Downloading Files</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="change_credentials_button">
       <property name="text">
        <string>Credentials</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
